browser-interface/packages/shared/comms/sagas.ts

```diff
@@ -370,18 +373,36 @@ function getInformationForCommsProfileRequest(state: RootState) {
   }
 }
 
-function stripSnapshots(profile: Avatar): Avatar {
+const snapshotsCache: Map<string, string> = new Map()
```

Comment:
feel free to ignore this comment because I'm not 100% how this works, but couldn't this cache get out of hands? entries won't be ever evevicted AFAIU, maybe use lru-cache instead?

Reply:
This should not be a critical case since the items added are linked to an effective change in profile and the values stored are four strings per change (a key base64 and a string for body and face256).

I'll come back to this issue though, in a separate PR. Thanks!